src/CommunityToolkit.Maui.Core/CommunityToolkit.Maui.Core.csproj

```diff
@@ -0,0 +1,48 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0;net6.0-android;net6.0-ios;net6.0-maccatalyst</TargetFrameworks>
```

Comment:
Let's try to keep target framework net6.0 only

Reply:
I can't see a good reason to not target platform code here, even this one being a Core project we may need to have platform code, .NET MAUI does that as I referenced in the PR's issue.

Reply:
It's clear that there is confusion around what `CommunityToolkit.Maui.Core` is supposed to be.

Let's please pause this PR and return to the [Proposal](https://github.com/CommunityToolkit/Maui/issues/213)  to define `CommunityToolkit.Maui.Core`.

Reply:
It depends on what we put into Core. If there is no platform specific code, we do not need to add specific target framework.
1. It reduces the package size
2. It simplifies dependencies
3. There is no confusion where to put platform specific code.